lib/y2_installbase.pm

```diff
@@ -264,18 +291,37 @@ sub save_strace_gdb_output {
 
 sub post_fail_hook {
     my $self = shift;
-    $self->SUPER::post_fail_hook;
-    get_to_console;
-    $self->detect_bsc_1063638;
-    $self->get_ip_address;
-    $self->remount_tmp_if_ro;
-    # Avoid collectin logs twice when investigate_yast2_failure() is inteded to hard-fail
-    $self->save_upload_y2logs unless get_var('ASSERT_Y2LOGS');
-    return if is_caasp;
-    $self->save_system_logs;
-
-    # Collect yast2 installer  strace and gbd debug output if is still running
-    $self->save_strace_gdb_output;
+    if (check_var("REMOTE_CONTROLLER", "vnc")) {
+        wait_screen_change { send_key 'ctrl-alt-shift-x' };
+        $self->save_remote_upload_y2logs unless get_var('ASSERT_Y2LOGS');
```

Comment:
As we cannot get things from the remote machine on the serial port, how all the `script_run` are going to work?
It's alright to ignore the errors in this case, but we will waste time on waiting for the return command. Also, `can_upload_logs` won't work properly as we run ping command which will fail and trigger network recovery.

Reply:
you can see how it works/looks here http://aquarius.suse.cz/tests/1316#step/welcome/13.
 in the logs tab you can see the upload tar.
script_run will fail but the commands will run and it will upload the logs as it is just a curl cmd